frappe/email/doctype/email_domain/email_domain.py

```diff
@@ -121,3 +63,26 @@ def on_update(self):
 				frappe.msgprint(
 					_("Error has occurred in {0}").format(email_account.name), raise_exception=e.__class__
 				)
+
+	@handle_error(_("Incoming email server incorrect"), _("Error connecting via IMAP/POP3: {e}"))
```

Comment:
sus

Reply:
reverted back to event parameter for the decorator + wrapped the translatable strings in a function